Python/ceval.c

```diff
@@ -442,7 +442,11 @@ _Py_InitializeRecursionLimits(PyThreadState *tstate)
     _tstate->c_stack_soft_limit = _tstate->c_stack_hard_limit + PYOS_STACK_MARGIN_BYTES;
 #else
     uintptr_t here_addr = _Py_get_machine_stack_pointer();
-#  if defined(HAVE_PTHREAD_GETATTR_NP) && !defined(_AIX) && !defined(__NetBSD__)
+/// XXX musl supports HAVE_PTHRED_GETATTR_NP, but the resulting stack size
+/// (on alpine at least) is much smaller than expected and imposes undue limits
+/// compared to the old stack size estimation.
```

Comment:
```suggestion
// gh-131338: musl supports HAVE_PTHRED_GETATTR_NP, but the resulting stack size
// (on Linux Alpine at least) is much smaller than expected and imposes undue limits
// compared to the old stack size estimation.
```

You may mention that `__GLIBC__` is used to detect musl support.

Reply:
Updated comment.  If that clarification doesn't feel like enough I'll rewrite the whole comment.